### PR TITLE
Rp story check

### PR DESCRIPTION
**Pull request by perryr16**
bug fixes from story double check

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -165,7 +165,7 @@ tr:nth-child(even) {
   max-width:15%;
   max-height:15%;
 }
-.thumnail {
-  height: 5vh;
-  width: 5vw;
+.thumbnail {
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:168]: spelling. this works nicely now

File: app/views/merchant/dashboard/show.html.erb [hunk 2]
```diff
@@ -16,8 +16,8 @@
   <section class="grid-item" id='order-<%=order.id%>'>
     <p>Order #: <%=link_to order.id, "/merchant/orders/#{order.id}" %> </p>
     <p>Date: <%= order.created_at %></p>
-    <p>Total Items: <%= order.total_items %></p>
-    <p>Total Value: <%= order.grandtotal %></p>
+    <p>Total Merchant Items: <%= @user.merchant.my_items_this_order(order.id).count %></p>
+    <p>Total Merchant Value: <%= @user.merchant.my_total_this_order(order.id) %></p>
   </section>
```
Comment (perryr16) [inline app/views/merchant/dashboard/show.html.erb:21]: some really fun new model methods

File: app/views/user_orders/show.html.erb [hunk 3]
```diff
@@ -43,7 +44,7 @@
   <% @order.item_orders.each do |item_order|%>
     <tr>
     <section id = "item-<%=item_order.item_id%>">
-        <td><img class="item-<%=item_order.item_id%>-image" src="item_order.item.image" alt="<%= item_order.item.name%> image"></td>
+        <td><img class="thumbnail" src=<%=item_order.item.image%> alt="<%= item_order.item.name%> image"></td>
```
Comment (perryr16) [inline app/views/user_orders/show.html.erb:47]: changed this class to a style, the spec now checks `has_css("img*=...")`

File: spec/models/merchant_spec.rb [hunk 4]
```diff
@@ -82,5 +82,47 @@
...
+
+      ItemOrder.create(order_id: @order1.id, item_id: @item1.id, status: 0, price: @item1.price, quantity: 1)
+      ItemOrder.create(order_id: @order1.id, item_id: @item2.id, status: 0, price: @item2.price, quantity: 1)
+      ItemOrder.create(order_id: @order1.id, item_id: @item3.id, status: 1, price: @item3.price, quantity: 1)
+      
+      expect(@merchant1.my_items_this_order(@order1.id)).to eq([@item1, @item2])
+      expect(@merchant2.my_items_this_order(@order1.id)).to eq([@item3])
+    end
+
+    it "my_total_this_order" do
```
Comment (perryr16) [inline spec/models/merchant_spec.rb:105]: model methods!

File: app/views/user_orders/show.html.erb (hunk 3 above)
Comment (foleymichelle9) [reply app/views/user_orders/show.html.erb:47]: Nice!

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (Mariana-21) [reply app/assets/stylesheets/application.css:168]: Nice catch

File: spec/models/merchant_spec.rb (hunk 4 above)
Comment (Mariana-21) [reply spec/models/merchant_spec.rb:105]: MODEL METHODS!!!!!

**Comment by foleymichelle9** [comment]
looks good!

**Pull request merged**